app/controllers/concerns/sufia/files_controller_behavior.rb

```diff
@@ -64,6 +64,7 @@ def load_resource_from_solr
 
     # routed to /files/new
     def new
+      set_variables_for_new_form
```

Comment:
Should we also be checking to make sure that current_user .can? edit collection? That'd be a good test to add.